SudoScript.Core/Tokenizer.cs

```diff
@@ -47,8 +47,6 @@ public sealed class TokenStream : IDisposable
 
     private char? _carry;
 
-    private readonly string _src;
```

Comment:
Pretty sure this was supposed to be used to exceptions. I would probably rather that we implement those exceptions than remove the property.